packages/api/app/Models/Traits/Params.js

```diff
@@ -57,27 +57,28 @@ class Params {
 			],
 		};
 
-		Model.queryMacro('withParams', async function withParams(
-			request,
-			options = { filterById: true },
-		) {
+		Model.queryMacro('withParams', async function withParams(request, options = {}) {
 			const { id, embed, page, perPage, order, orderBy, ids, notIn } = request.params;
 
 			// eslint-disable-next-line no-underscore-dangle
 			const resource = this._single.table;
 
+			const { filterById = true, skipRelationship = [] } = options;
```

Comment:
```suggestion
			const { filterById = true, skipRelationships = [] } = options;
```
Justificativa: é uma array 🙂

Reply:
Cirurgico como sempre @mateus4k :1st_place_medal: